testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/OAuthClient.java

```diff
@@ -1175,10 +1175,10 @@ public UserInfo doUserInfoRequest(String accessToken) {
         }
     }
 
-    public UserInfoResponse doUserInfoRequestByGet(String accessToken) throws Exception {
+    public UserInfoResponse doUserInfoRequestByGet(OAuthClient.AccessTokenResponse accessTokenResponse) throws Exception {
         try (CloseableHttpClient client = HttpClientBuilder.create().build()) {
             HttpGet get = new HttpGet(getUserInfoUrl());
-            get.setHeader("Authorization", "Bearer " + accessToken);
+            get.setHeader("Authorization", accessTokenResponse.getTokenType() + " " + accessTokenResponse.getAccessToken());
             if (dpopProof != null) {
                 get.addHeader("DPoP", dpopProof);
```

Comment:
`"DPoP"` might be `TokenUtil.TOKEN_TYPE_DPOP`.

Reply:
L1183 ` get.addHeader("DPoP", dpopProof);`
to
`get.addHeader(TokenUtil.TOKEN_TYPE_DPOP, dpopProof);`
But, you need not to modify it.